distribution/src/scripts/integrator.bat

```diff
@@ -1,181 +1,192 @@
-@echo off
-
-REM ---------------------------------------------------------------------------
-REM        Copyright 2005-2009 WSO2, Inc. http://www.wso2.org
-REM
-REM  Licensed under the Apache License, Version 2.0 (the "License");
-REM  you may not use this file except in compliance with the License.

```

Comment:
The file has a diff that is not related to your fix. Shall we only have the relevant diff?

Reply:
Done